bliss/simulator/simulated_dataset.py

```diff
@@ -188,11 +188,13 @@ def __init__(
         self,
         train_n_batches: int,
         batch_size: int,
+        bands: List,
         num_workers: int,
         cached_data_path: str,
         file_prefix: str,
         val_split_file_idxs: List[int],
         test_split_file_idxs: List[int],
+        z_score_transform: bool,
```

Comment:
Do we still need this if the z-scoring is now being done in the encoder?

Reply:
If we're set on z-scoring only being done in the encoder then this is definitely not needed. Haven't noticed a decrease in speed as of yet so think it should be fine to remove this.